docs/content.zh/docs/ops/metrics.md

```diff
@@ -28,26 +28,26 @@ under the License.
 
 # 指标
 
-Flink exposes a metric system that allows gathering and exposing metrics to external systems.
+Flink 公开了一个指标系统，允许收集和公开指标给外部系统。
 
-## Registering metrics
+## 注册指标
```

Comment:
missing link tag ?
The same case should be noted below.

Reply:
sorry, I can't find the link here, what's your mean?

Reply:
<a name="tegistering-metrics"></a>

You will get more relevant information in the translation part of the official documents.